docs/New-Maintainer-Checklist.md

```diff
@@ -25,12 +28,15 @@ A few requests:
   button in "create a merge commit" mode for Homebrew/brew or "squash and
   merge" for a single formulae change. If in doubt, check with e.g. GitX that
   you've not accidentally added merge commits
-- still create your branches on your fork rather than in the main repository
-- if still in doubt please ask for help and we'll help you out - these are
-  probably worth a read:
-    - https://github.com/Homebrew/brew/blob/master/docs/Brew-Test-Bot-For-Core-Contributors.md
-    - https://github.com/Homebrew/brew/blob/master/docs/Maintainer-Guidelines.md
-    - possibly everything else in the documentation
+- still create your branches on your fork rather than in the main repository.
+  Note GitHub's UI will create edits and reverts on the main repository if you
+  make edits or click revert on the Homebrew/brew repository rather than your
+  own fork.
```

Comment:
I think it might be better just to say not to use the gui for edits except on your own fork. In my experience branches build up and stick around for a while, and other maintainers don't know if they're safe to delete or not. It's not much of a hurdle to switch to your own form before using the web interface (and also not a huge deal if it happens accidentally).